src/main/java/frc/robot/util/AutoGenerator.java

```diff
@@ -62,7 +62,7 @@ public AutoGenerator(DrivetrainSubsystem drivetrainSubsystem, FloorArmSubsystem
         eventMap.put("scoreLow", new ScoreLow(drivetrainSubsystem, floorArmSubsystem, intake));
         eventMap.put("scoreConeHigh", new ExtendAndScoreCone(drivetrainSubsystem, floorArmSubsystem, armSubsystem, intake));
         eventMap.put("scoreCubeHigh", new RotateAndScoreCube(drivetrainSubsystem, floorArmSubsystem, armSubsystem, floorIntake));
-        eventMap.put("autoBalance", new PrintCommand("Auto balance begin"));
+        eventMap.put("autoBalance", new InstantCommand(drivetrainSubsystem::setBalanceTargetDefault, drivetrainSubsystem));
```

Comment:
We should add to our DefaultDriveCommand code to null the balanceTarget and translationTarget anytime any joystick is non-zero (similar to what we did for rotationTarget nulling